### PR TITLE
[1LP][RFR] Fix fill for orchestration catalog item view 

### PR DESCRIPTION
**Pull request by izapolsk**
where provider dropdown appears with some timeout

{{pytest: --long-running --use-template-cache --use-provider=azure -k test_provision_stack cfme/tests/services/test_provision_stack.py}}

### CODE REVIEW

File: cfme/services/myservice/ui.py [hunk 1]
```diff
@@ -179,10 +179,13 @@ class ReconfigureServiceView(SetOwnershipForm):
 
     @property
     def is_displayed(self):
+        # todo check whether this is Feature or Bug (it seems we need to pass parent item instead)
```
Comment (mshriver) [inline cfme/services/myservice/ui.py]: I think this is 1658906, please include in the comment or set name based on the BZ state

Comment (izapolsk) [reply cfme/services/myservice/ui.py]: I forgot that my PR hadn't been merged and asked @niyazRedhat to create another PR. 
I'll fix this here.

**Comment by mshriver** [comment]
PRT failures do not look related to the fixes/changes made here.

**Comment by izapolsk** [comment]
@mshriver, yep, it seems flaky tests.

**Pull request merged**